cmd/server/assets/login/_loginscripts.html

```diff
@@ -9,14 +8,12 @@
   });
 
   function setSession(user) {
-    let factorCount = user.multiFactor.enrolledFactors.length;
```

Comment:
This is available in the cookie now, we don't need to pull it from the javascript.